BackendBench/scripts/get_tests_stat.py

```diff
@@ -0,0 +1,274 @@
+    elif correct_or_perf == "performance":
+        stats = analysis_results["perf_stats"]
+        num_tests_str = "num_perf_tests"
+    else:
+        raise ValueError(f"Invalid value for 'correct_or_perf': {correct_or_perf}")
+
+    operations = analysis_results["operations"]
+
+    print(f"\n{'=' * 60}")
+    print(f"TEST STATISTICS SUMMARY FOR {suite_name.upper()} SUITE {correct_or_perf.upper()} TESTS")
+    print(f"{'=' * 60}")
+
+    print("\nOverall Statistics:")
+    print(f"  Total operations: {stats['total_operations']}")
+    print(f"  Total tests: {stats['total_tests']}")
+    print(f"  Average tests per operation: {stats['mean_tests']:.2f}")
+    print(f"  Median tests per operation: {stats['median_tests']:.2f}")
+    print(f"  Min tests per operation: {stats['min_tests']}")
+    print(f"  Max tests per operation: {stats['max_tests']}")
+    print(f"  Standard deviation: {stats['stdev_tests']:.2f}")
+
+    if operations:
+        test_counts = [info[num_tests_str] for info in operations.values()]
+        bins = [1, 5, 10, 25, 50, 100, float("inf")]
+        bin_labels = ["1-4", "5-9", "10-24", "25-49", "50-99", "100+"]
+        distribution = [0] * (len(bins))
+
+        count_1_tests = 0
+        for count in test_counts:
+            if count == 1:
+                count_1_tests += 1
+            for i in range(len(bins) - 1):
+                if bins[i] <= count < bins[i + 1]:
+                    distribution[i] += 1
+                    break
+
+        print("\nTest Count Distribution:")
+        print(f"  Operations with 1 test: {count_1_tests}")
+        for i, (label, count) in enumerate(zip(bin_labels, distribution)):
+            percentage = (count / len(operations)) * 100
+            print(f"  {label:>6} tests: {count:3d} operations ({percentage:5.1f}%)")
+
+
+@click.command()
+@click.option(
+    "--suite",
+    default="opinfo",
+    type=click.Choice(["smoke", "opinfo", "torchbench", "facto", "all"]),
+    help="Which suite to analyze",
+)
+@click.option(
+    "--ops",
+    default=None,
+    type=str,
+    help="Comma-separated list of ops to run",
+)
+@click.option(
+    "--topn-inputs",
+    "--topn",
+    default=None,
+    type=int,
+    help="Select the top N largest inputs for each op (default: all inputs)",
+)
+@click.option(
+    "--torchbench-data-path",
+    default=DEFAULT_HUGGINGFACE_URL,
+    type=str,
+    help="Path to TorchBench operator data",
+)
+def cli(
+    suite,
+    ops,
+    topn_inputs,
+    torchbench_data_path,
+):
+    if ops:
+        ops = ops.split(",")
+    suites_to_analyze = []
+
+    suite_dict = {
+        "smoke": lambda: SmokeTestSuite,
+        "opinfo": lambda: OpInfoTestSuite(
+            "opinfo_cuda_bfloat16",
+            "cuda",
+            torch.bfloat16,
+            filter=ops,
+        ),
+        "torchbench": lambda: TorchBenchTestSuite(
+            "torchbench",
+            torchbench_data_path,
+            filter=ops,
+            topn=topn_inputs,
```

Comment:
If you're just getting the suite and counting ops I don't think this is actually needed (and you wouldn't want it). I'd just remove this arg